src/ptrmask-integration.adoc

```diff
@@ -0,0 +1,15 @@
+[#section_pointer_masking_integration]
+== Integrating {cheri_base_ext_name} and {cheri_default_ext_name} with Pointer Masking
+
+The pointer masking extensions SSnpm, Sspm and Supm are compatible with {cheri_default_ext_name}.
```

Comment:
According to the RISC-V Pointer Masking spec, the existing Pointer Masking extensions are:
* Smmpm
* Smnpm
* Ssnpm

Could we align this text with the contents of the RISC-V spec?

Reply:
I took the names from https://github.com/riscv/riscv-profiles/blob/main/src/rva23-profile.adoc

Reply:
```suggestion
The pointer masking extensions Smmpm, Smnpm, SSnpm, Sspm and Supm are compatible with {cheri_default_ext_name}.
```